packages/react-components/src/Concepts/Icons/ReactIconGrid.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { FluentIconsProps } from '@fluentui/react-icons';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import * as ReactIcons from '@fluentui/react-icons';
+import { Input } from '@fluentui/react-input';
+import { makeStyles } from '@fluentui/react-make-styles';
+
+const useStyles = makeStyles({
+  grid: theme => ({
+    width: '100%',
+    marginLeft: 'auto',
+    boxSizing: 'border-box',
+    marginRight: 'auto',
+    display: 'block',
+    paddingLeft: '16px',
+    paddingRight: '16px',
+    paddingTop: '10px',
+    fontFamily: 'Arial',
+    maxWidth: '105ch',
+    position: 'relative',
+
+    '& span': {
```

Comment:
You should probably limit this to spans which are immediate children.
```suggestion
    '> span': {
```